packages/admin/src/form/Input.ts

```diff
@@ -0,0 +1,13 @@
+import { Theme } from "@material-ui/core";
+
+/**
+ * @deprecated Getting the default-input-styles should no longer be necessary, just use CometAdminInputBase instead of the material-ui InputBase
```

Comment:
This change goes to the next-branch (incompatible label) and will be a major release. Should getDefaultVPAdminInputStyles still be supported in the next major release? If not, it can be removed without deprecation.

Reply:
It's currently still used in some Components (e.g. ColorPicker).  
I will remove it, in a seperate PR, as soon as it is no longer used anywhere.